pyproject.toml

```diff
@@ -74,3 +74,34 @@ requires = [
         directory = "change"
         name = "Changes"
         showcontent = true
+
+
+[tool.cibuildwheel]
+skip = "cp36-* cp37-* pp37-* pp38-macosx* *-manylinux_i686 *_ppc64le *_s390x *-musllinux*"
+build-verbosity = "3"
+before-build = "bash {project}/tools/wheels/cibw_before_build.sh {project}"
+before-test = "pip install -r {project}/test_requirements.txt"
+test-command = "bash {project}/tools/wheels/cibw_test_command.sh {project}"
+
+[tool.cibuildwheel.linux]
+manylinux-x86_64-image = "manylinux2014"
+manylinux-aarch64-image = "manylinux2014"
+environment = { CFLAGS="-std=c99 -fno-strict-aliasing", LDFLAGS="-Wl,--strip-debug", OPENBLAS64_="/usr/local", NPY_USE_BLAS_ILP64="1", RUNNER_OS="Linux" }
+
+[tool.cibuildwheel.macos]
+# TODO: Add universal2 wheels, we need to fuse them manually
```

Comment:
I think we will not be building universal2 wheels. See the discussion in #20787 and the [scientific-python discord discussion](https://discuss.scientific-python.org/t/should-the-scientifc-python-community-provide-universal2-wheels/236)

```suggestion
# For universal2 wheels, we will need to fuse them manually
```

Reply:
yep, I came across those links after creating this PR where I just copy-pasted the original comment here. I'll just commit your suggestion then.